ArmTemplates/Application-Insights/metric-alerts.json

```diff
@@ -115,10 +122,11 @@
                         {
                             "name": "metric critria",
                             "metricName": "[parameters('metricName')]",
-                            "dimensions": [],
+                            "dimensions": "[parameters('dimensions')]",

```

Comment:
This enables us to pass in something like:

```
[
  {  
     "name": "StatusCode",
     "operator": "Include",
     "values": [  "429"  ]
   }
]
```
to limit the queries to only return requests with a status code of 429